web/annotation_layer_builder.css

```diff
@@ -55,6 +61,10 @@
   transform-origin: 0 0;
 }
 
+.annotationLayer .linkAnnotation {
+  border: var(--link-border);
```

Comment:
Also, given that a `linkAnnotation` can specify a custom border style/color would it perhaps be more correct (or at least safer) to utilize an `outline` here instead?